bin/taiko.js

```diff
@@ -141,11 +147,12 @@ if (isTaikoRunner(process.argv[1])) {
                         });
                     });
                 } else {
-                    runFile(fileName, observe, program.waitTime);
+                    runFile(taiko, fileName, observe, program.waitTime);
                 }
             } else {
+                loadPlugins(program.plugin);
```

Comment:
loading plugins may not be required as `--plugins` will add them to env variables and then `require('taiko')` takes care of loading them